gobblin-metastore/src/main/java/gobblin/metastore/testing/ITestMetastoreDatabase.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metastore.testing;
```

Comment:
In general, we follow the pattern that any new classes created just for testing purposes still go under `src/test`. What dependency problems are you facing? You can also add more dependencies in the `build.gradle` using `testCompile`